app/src/main/java/org/wikipedia/edit/EditSectionActivity.kt

```diff
@@ -325,6 +328,20 @@ class EditSectionActivity : BaseActivity(), ThemeChooserDialog.Callback, EditPre
         BreadCrumbLogEvent.logInputField(this, editSummaryFragment.summaryText)
     }
 
+    private fun getEditTags(): List<String> {
+        return buildList {
```

Comment:
Curious about in what condition will multiple tags be output?

Reply:
That's fair -- I updated it to produce a single tag.